### PR TITLE
support Airflow 2.8.0

### PR DESCRIPTION
**Pull request by bobelev**
Airflow 2.8.0 has been released https://github.com/apache/airflow/releases/tag/2.8.0

Let's make this plugin compatible

### CODE REVIEW

**Comment by simpl1g** [comment]
@bryzgaloff Hi, is it possible to review this PR? Would be great to bump version and update airflow version 😃

**Comment by bryzgaloff** [comment]
Hi guys, thank you for bringing the PR up. As you may have noticed, it did not pass some tests, so I was not able to merge it.

However, the tests were failing due to some versions inconsistencies in the packages installed during the tests execution. I have fixed it in #74. That PR is now merge, so please rebase yours onto the latest master: it will apply the versions fix and all the older tests should be passing after that.

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@ jobs:
     strategy:
       matrix:
         python-version: ["3.8", "3.9", "3.10", "3.11"]
-        airflow-version: ["2.0.2", "2.1.4", "2.2.5", "2.3.4", "2.4.3", "2.5.3", "2.6.3", "2.7.1"]
+        airflow-version: ["2.0.2", "2.1.4", "2.2.5", "2.3.4", "2.4.3", "2.5.3", "2.6.3", "2.7.1", "2.8.0"]
```
Comment (bryzgaloff) [inline .github/workflows/tests.yml:29]: Let's update to 2.8.1 now, it is the latest patch release of Airflow currently

**Review by bryzgaloff — COMMENTED** [review]
All is good, just requires a rebase to make sure all the tests are passing

**Comment by bobelev** [comment]
Hi! Rebased, all green.

**Comment by bryzgaloff** [comment]
Thank you, release v1.2.0 is being published: https://github.com/bryzgaloff/airflow-clickhouse-plugin/releases/tag/v1.2.0

**Comment by bryzgaloff** [comment]
Unfortunately, there was an issue publishing the pipeline, so the latest release is not yet available: #75

I am on it, some time is needed to fix it.

**Pull request merged**